src/oss-download/DownloadTool.cs

```diff
@@ -5,8 +5,10 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Security.Cryptography;
```

Comment:
I don't see what requires this in this file.

Reply:
there were unwanted usings in various files...removed